src/components/SignUpForm/index.tsx

```diff
@@ -0,0 +1,45 @@
+import ButtonLarge from '../shared/ButtonsLarge';
+import Logo from '../shared/Icons/logo';
+import InputField from '../shared/InputFields';
+import Link from '../shared/Links/links';
+
+const SignUpForm = () => {
+  return (
+    <div className='flex-col box-content items-center h-[34.75rem] w-[22.5rem] bg-indigo border-4m flex bg-white rounded-lg mt-16 ml-[7.438rem]'>
```

Comment:
It's better to avoid using Arbitrary values when is not necessary for example in `ml-[7.438rem]`